ORST/Assets/Scripts/Core/Interactions/ForbiddenSpace/HandHeadProximity.cs

```diff
@@ -0,0 +1,91 @@
+using System;
+using System.Collections.Generic;
+using UnityEngine;
+
+namespace ORST.Core.Interactions
+{
+    public class HandHeadProximity : MonoBehaviour {
+        public event Action ForbiddenSpaceEntered;
+        public event Action ForbiddenSpaceExited;
+        public event Action<float> ForbiddenSpaceUpdated;
+
+        [SerializeField] private Collider m_InnerCollider;
+        [SerializeField] private Collider m_OuterCollider;
+
+        private readonly List<Transform> m_Intersector = new ();
+        private GameObject m_ForbiddenGameObject;
+
+        private void Update() {
+            ForbiddenSpaceUpdated?.Invoke(GetClosestIntersectLinear());
+        }
+
+        private float GetClosestIntersectLinear() {
+            if (m_Intersector.Count <= 0) {
+                return 0;
```

Comment:
Always use the form `#.#f` for floats
```suggestion
                return 0.0f;
```